examples/federated_learning/yolov5_coco128_mistnet/interface.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from sedna.algorithms.aggregation import MistNet
+from sedna.algorithms.client_choose import SimpleClientChoose
```

Comment:
Sedna-related import are not recommended in `interface`.

Reply:
Why not recommend it?

Reply:
`interface`  focus on the application itself (run independently without sedna). And, the deployer cannot change the algorithm selection by CRD config.

Reply:
Kept as Jie Pu suggested.